Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.py

```diff
@@ -0,0 +1,40 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+def is_valid_attack_pattern(items):
+
+    try:
+        results = demisto.executeCommand('mitre-get-attack-pattern-value', {'attack_ids': items})
+        list_contents = results[0]['Contents']
+        values = [content.get('value') for content in list_contents]
+        return values if values else False
+
+    except ValueError as e:
+        if 'verify you have proper integration enabled to support it' in str(e):
```

Comment:
`else` statement is missing. if this is a diffrent error, it should be logged as error with its message

Reply:
done